tests/test_prolific.py

```diff
@@ -11,6 +11,7 @@
     "estimated_completion_time": 5,
     "external_study_url": "https://www.example.com/ad?recruiter=prolific&PROLIFIC_PID={{%PROLIFIC_PID%}}&STUDY_ID={{%STUDY_ID%}}&SESSION_ID={{%SESSION_ID%}}",
     "internal_name": "fake experiment title (TEST_EXPERIMENT_UID)",
+    "is_custom_screening": True,
```

Comment:
`is_custom_screening` has been added to the fixtures, but it would have been good to add a test that ensures that this variable propagates from config to `draft_study`. Let's not worry about it for now though.

Reply:
I added an assertion to the `test_can_create_a_draft_study_and_delete_it`  test verify it is indeed set to `True`.